frontend/online-shop/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
Do you still use those classes? If not you can clean this `.css` folder

Reply:
You are right, fixed